src/exceptions/api.exception.ts

```diff
@@ -2,11 +2,13 @@ export class ApiException extends Error {
   constructor(
     public message: string,
     public status?: number,
-    public body?: string | FormData
+    public body?: string | FormData,
+    public tag?: string
```

Comment:
What's "tag" ?

Reply:
Tag is the tag you give to the i18next to replay... While body gives us more information it shouldn't give us the error message to replay to the user as all that is fixed strings.